pkg/embeddedcluster/util.go

```diff
@@ -52,3 +57,35 @@ func ClusterID(client kubernetes.Interface) (string, error) {
 
 	return configMap.Data["embedded-cluster-id"], nil
 }
+
+// ClusterConfig will get the list of installations, find the latest installation, and get that installation's config
+func ClusterConfig(ctx context.Context) (*embeddedclusterv1beta1.ConfigSpec, error) {
+	clientConfig, err := k8sutil.GetClusterConfig()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get cluster config: %w", err)
+	}
+
+	scheme := runtime.NewScheme()
+	embeddedclusterv1beta1.AddToScheme(scheme)
+
+	kbClient, err := kbclient.New(clientConfig, kbclient.Options{
+		Scheme: scheme,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("failed to get kubebuilder client: %w", err)
+	}
+
+	var installationList embeddedclusterv1beta1.InstallationList
+	err = kbClient.List(ctx, &installationList, &kbclient.ListOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to list installations: %w", err)
+	}
+
```

Comment:
are there scenarios where the list returned could be empty?

Reply:
There's an installation created with every embedded-cluster install, so ideally no